intro-to-R-tidyverse/03-intro_to_tidyverse.Rmd

```diff
@@ -8,15 +10,24 @@ editor_options:
   chunk_output_type: inline
 ---
 
-**CCDL 2020**
 
-## Objective for this notebook analysis: 
+## Objectives
+
+This notebook will demonstrate how to:
+
+- Use the tidyverse package to read and write data frames
```

Comment:
```suggestion
- Use functions from the tidyverse to read and write data frames
```

Reply:
All sounds good! Will go ahead and do that.